libsrc/gamate/vga.s

```diff
@@ -1,6 +1,11 @@
-
 ; VGA charset for the Gamate conio implementation
 
+.export fontdata
+
+.rodata
```

Comment:
Both .export and .rodata should be indented.

Reply:
Done